lib/ui/painting.dart

```diff
@@ -860,6 +860,64 @@ class Path extends NativeFieldWrapperClass2 {
   void _arcTo(double left, double top, double right, double bottom,
               double startAngle, double sweepAngle, bool forceMoveTo) native "Path_arcTo";
 
+  /// Appends up to four conic curves weighted to describe an oval of [radius]
+  /// and rotated by [xAxisRotation].
+  ///
+  /// The first curve begins from the last point in the path and the last ends
+  /// at [arcEnd]. The curves follow a path in a direction determined by
+  /// [isClockwiseDirection] and [isLargeArc] in such a way that the sweep angle
+  /// is always less than 360 degrees.
+  ///
+  /// A simple line is appended if either either radii are zero or the last
+  /// point in the path is [arcEnd]. The radii are scaled to fit the last path
+  /// point if both are greater than zero but too small to describe an arc.
+  ///
+  void arcToPoint(Offset arcEnd, {
+      Radius radius : Radius.zero,
+      double xAxisRotation : 0.0,
+      bool isLargeArc : false,
+      bool isClockwiseDirection : true,
+    }) {
```

Comment:
We prefer using `isFoo` for properties, rather than arguments.
Also I think "direction" is probably redundant, so in the interests of making the name shorter, we should probably omit that.
Is the rotation argument any more "xAxis" than "yAxis"? I mean, technically it's rotation around the z-axis, right? I think just "rotation" (or "angle") is sufficient.
Finally for the clockwise argument, we generally prefer to default booleans to false, but if think the default should be clockwise then that's fine (it's more important for the argument to be named positively than for the default to be false).

nit: unindent all the arguments by two, and no space before the colon.

Reply:
* Renamed the `isFoo` args.
* Got rid of the "direction" redundancy.
* Renamed argument to "rotation".
* Skia tends to default to clockwise for other calls so I wanted to be consistent with that and `anticlockwise: false` seemed awkward to read.
* Unindented and got rid of the spaces.